packages/apps/plugins/plugin-table/src/components/ObjectTable/ObjectTable.stories.tsx

```diff
@@ -28,6 +28,15 @@ const Story = () => {
     generator.addSchemas();
     generator.createObjects({ [TestSchemaType.project]: 6 });
 
+    // TODO(zan): Currently we need to cast as any since `_graph` is marked @internal.
+    if (!(client as any)._graph.types.isEffectSchemaRegistered(TableType)) {
+      (client as any)._graph.types.registerEffectSchema(TableType);
+    }
+
+    // We need a table to reference
+    // TODO(zan): Workout how to get this to not double add in debug.
+    space.db.add(create(TableType, { title: 'Other table', props: [], schema: generator.schemas[3] }));
```

Comment:
Nice, thanks for that.

Reply:
That doesn't work out because we need to run the following first with access to client.

```
 // TODO(zan): Currently we need to cast as any since `_graph` is marked @internal.
    if (!(client as any)._graph.types.isEffectSchemaRegistered(TableType)) {
      (client as any)._graph.types.registerEffectSchema(TableType);
    }
```

Reply:
ah, we can update the callback so it has access to client, but fine to leave as a todo for now